.github/workflows/workflows-simple-canary-prod-test.yaml

```diff
@@ -0,0 +1,30 @@
+name: workflows-simple-canary-prod-test
+'on':
+  schedule:
+    - cron: "0 * * * *"  # run hourly
+  workflow_dispatch:
+  pull_request: #remove this before merging
+    branches: [ develop ]
+
+jobs:
+  run-simple-canary-prod-test:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    steps:
+      - name: Dispatch to terra-github-workflows
+        uses: broadinstitute/workflow-dispatch@v3
+        with:
+          workflow: workflows-canary-prod-test
+          repo: broadinstitute/terra-github-workflows
+          ref: refs/heads/sps_canary_test # TODO: change to refs/heads/main before merging
```

Comment:
Once https://github.com/broadinstitute/terra-github-workflows/pull/208 merges I will update this to main before merging this PR.